### PR TITLE
Mobile core signal

### PR DESCRIPTION
**Pull request by calebk1**
https://jira.corp.adobe.com/browse/MOB-16041

### CODE REVIEW

File: src/pages/documentation/mobile-core/signal/tabs/api-reference/register-extension/ios-aep.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This API no longer exists in `Signal`. Instead, the extension should be registered by calling the `registerExtensions` API in the MobileCore. Please see the updated SDK initialization steps at the [migrate to Swift tutorial.](../../resources/migrate-to-swift.md#update-sdk-initialization) FIX LINK
```
Comment (spoorthipujariadobe) [inline src/pages/documentation/mobile-core/signal/tabs/api-reference/register-extension/ios-aep.md:1]: Does this link need to be fixed?

File: src/pages/documentation/mobile-core/signal/event-reference.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# Signal event reference
+
+## Events handled
+
+The following events are handled by the Signal extension:
+
+### Rules Engine response content
+
+This event is used by the Signal extension to queue and send network calls that correspond to the [Sync PII Consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#sync-pii-consequence) or postback message. FIX LINK
```
Comment (spoorthipujariadobe) [inline src/pages/documentation/mobile-core/signal/event-reference.md:9]: Does link needs fixing?

File: src/pages/documentation/mobile-core/signal/tabs/api-reference/register-extension/ios-aep.md (hunk 1 above)
Comment (calebk1) [reply src/pages/documentation/mobile-core/signal/tabs/api-reference/register-extension/ios-aep.md:1]: Me putting FIX LINK (like here) is generally for ones where I don't know exactly where it'll live on the new-side just yet (which is why I have a reminder for me to do it later).

File: src/pages/documentation/mobile-core/signal/event-reference.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+#### Event data example
+
+The following message is an example of a postback:
+
+```json
+{
+    "id": "9d40f5665d5bdbe96dcb3a24f4e4fe98d686a602",
+    "type": "pb",
+    "detail": {
+        "templateurl": "https://www.endpoint.com/post/{%urlenc(~sdkver)%}",
```
Comment (spoorthipujariadobe) [inline src/pages/documentation/mobile-core/signal/event-reference.md:38]: `templateUrl` from original docs differ from the one here
`"templateurl"   : "https://www.endpoint.com/post/<div data-gb-custom-block data-tag="urlenc"></div>",`

**Review by spoorthipujariadobe — APPROVED** [review]
LGTM, just one small correction

**Pull request merged**